docs/platform-integration/essentials/email.md

```diff
@@ -39,16 +52,10 @@ In iOS 9 and greater, Apple enforces what schemes an application can query for.
 
 # [Windows](#tab/windows)
 
-No platform differences.
+No additional setup required.
 
 -----
-
-## Get started
-
-[!INCLUDE [get-started](includes/get-started.md)]
-
-> [!TIP]
-> To use the Email API on iOS you must run it on a physical device, else an exception will be thrown.
+<!-- markdownlint-enable MD025 -->
 
 ## Using Email
```

Comment:
Looking at the preview (https://review.docs.microsoft.com/en-us/dotnet/maui/platform-integration/essentials/email?branch=pr-en-us-109&tabs=android), the doc twice tells us to import the Essentials namespace. Once at the beginning, then again under Using email.

Reply:
Also, in the preview there's a sentence that needs more words:

"The Email functionality works by calling the ComposeAsync method an EmailMessage that contains information about the email:"

Should be something like:

"The Email functionality works by calling the ComposeAsync method with an EmailMessage argument that contains information about the email:"